pax-chassis-web/src/lib.rs

```diff
@@ -85,6 +85,7 @@ impl PaxChassisWeb {
         let manifest = Self::fetch(&format!("http://localhost:9000/create/load{query_string}"))
             .await
             .expect("failed to fetch manifest from remote");
+        // let manifest = serde_json::from_str(&pax_cartridge::INITIAL_MANIFEST).unwrap();
```

Comment:
Please remove things rather than comment in the future. We have commit history if we need it.